### PR TITLE
Added Logs To Sell Vehicle To Customer

### PR DESCRIPTION
**Pull request by djontop**
Here We Added Logs To Sell Vehicle To Customers. When Employee Sell Vehicle To Customer Log Will Be Created ❣️
image

### CODE REVIEW

**Comment by djontop** [comment]
Personally Verified Everything ❣️

**Comment by GodyFromDiscord** [comment]
A log for this already exists under qb/smallresources/server/logs.lua 

    ['vehicleshop'] = '',

**Comment by djontop** [comment]
> vehicleshop

Its Different I Added Logs To Vehicle Shop Not Vehicle Sale. log Added in qb/smallresouces/server/logs is of vehiclesale with is when player sale his owned vehicle that logs created . But I added Logs to When you purchase vehicle from PDM From sales man

File: server.lua [hunk 1]
```diff
@@ -306,6 +306,7 @@ RegisterNetEvent('qb-vehicleshop:server:sellShowroomVehicle', function(data, pla
         local vehicle = data
         local vehiclePrice = QBCore.Shared.Vehicles[vehicle]['price']
         local commission = round(vehiclePrice * Config.Commission)
+        local bossmenu = round(vehiclePrice * Config.Commissionboss)
```
Comment (gigitygamer) [inline server.lua:309]: This needs to be local bossmenu = round(vehiclePrice * Config.bossmenu) - per your changes to the config.lua in this pull

**Pull request closed without merging**